pkg/controller/compliance/compliance_controller_test.go

```diff
@@ -151,18 +151,6 @@ var _ = Describe("Compliance controller tests", func() {
 		}, &dpl)).NotTo(HaveOccurred())
 		Expect(dpl.Spec.Template.ObjectMeta.Name).To(Equal(render.ComplianceServerName))
 
-		By("changing the cluster type to Managed")
-		Expect(c.Update(
```

Comment:
This update fails with no resourceVersion. Also the change to Managed is no longer done in the Installation so this Update is not needed, hence the removal.